service/src/main/java/com/codeforcommunity/processor/ReservationProcessorImpl.java

```diff
@@ -130,7 +131,7 @@ private void blockReservedCheck(int blockId, int userId) {
     }
 
     // neither the user nor a team they are on reserved the block, so they did not reserve it
-    throw new IncorrectBlockStatusException(blockId, "reserved");
+    throw new IncorrectBlockStatusException(blockId, Collections.singletonList(ReservationAction.INCOMPLETE));
```

Comment:
Why would this one be Incomplete but the other exceptions in this method are Reserve?